src/uption.rs

```diff
@@ -41,7 +42,7 @@ impl Uption {
         export_scheduler
             .join()
             .expect("The export scheduler thread has panicked");
-        println!("Uption stopped");
+        error!("Uption stopped");
```

Comment:
This could be just info level since we already log error in the sub-threads. Later we might also have functionality to stop Uption without any error occurring (for example if configuration is changed).
```suggestion
        info!("Uption stopped");
```